ols.py

```diff
@@ -20,8 +20,7 @@
 
 load_dotenv()
 
-instruct_model = os.getenv("INSTRUCT_MODEL", "ibm/granite-13b-instruct-v1")
-rag_model = os.getenv("RAG_MODEL", "ibm/granite-13b-chat-v1")
+base_completion_model = os.getenv("BASE_COMPLETION_MODEL", "ibm/granite-20b-instruct-v1")
```

Comment:
Why do we want to make it a global variable when it is only used one function?

Reply:
long term we almost certainly don't(i doubt that function sticks around either).  but for now i was trying to minimize the changes to the overall code itself while adding in the configurability bits.